wiki/Tournaments/Official_support/en.md

```diff
@@ -77,93 +131,41 @@ Profile badge images must adhere to the following standards:
 
 Please consult the following examples of approved badges:
 
-![](https://assets.ppy.sh/profile-badges/flc2-2020@2x.png)
-![](https://assets.ppy.sh/profile-badges/irt2-2020@2x.png)
+![](https://assets.ppy.sh/profile-badges/ysc4-2021@2x.png) ![](https://assets.ppy.sh/profile-badges/perennial-2021@2x.png) ![](https://assets.ppy.sh/profile-badges/timeskip-2021@2x.png)
 
 Badge images that do not meet these standards will be refused with given reasoning, and you may be asked to completely redesign your badge in some cases. This is fairly rare, but should be accounted for.
 
-## Requesting main-menu banner support
+In rare cases, the osu!team may choose to award exceptional tournaments that go above and beyond the expected standards with badges for the top three finalists. This is done at the discretion of the osu!team - you will be contacted if your tournament is deemed worthy.
+
+### Requesting main-menu banner support
 
 In rare circumstances, some long-running community tournaments may qualify for in-game main-menu banner support. It's at the osu! team's discretion to decide if your tournament can be advertised in-game, as there are many factors to consider. The minimum requirements to request in-game advertising are as follows:
 
-- Your tournament meets the [criteria for applicable tournaments](#criteria)
-- You have successfully run *at least* two tournaments in the past
-- The live streams of your current tournament average over 100 concurrent viewers
-- You are able to provide the schedule of the matches and live streams ahead of time
+- Your tournament meets the [eligibility criteria for applicable tournaments](#eligibility).
+- You have successfully run *at least* two tournaments in the past.
+- The live streams of your current tournament average over 100 concurrent viewers.
+- Your tournament is nearing its conclusion (e.g. your tournament is at Quarterfinals or above and it started on Round of 32).
+- You are able to provide the schedule of the matches and live streams ahead of time.
 
 If you satisfy the above criteria, you may make a request for main-menu banner support by sending an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh). The banner must adhere to the following standards:
 
 - The banner image must be a PNG.
 - The banner image must be **1500x256**px in dimension.
```

Comment:
This is pending confirmation, but there is a certain leniency for banner sizes

Reply:
Confirmed. 1500x256px is the standard.
The discrepancy in size comes from scaling/edits to the banner from the osu!team, and not from user submission.